airbyte-api/src/main/openapi/config.yaml

```diff
@@ -1583,6 +1583,10 @@ components:
           $ref: "#/components/schemas/DestinationDefinitionSpecification"
         jobInfo:
           $ref: "#/components/schemas/SynchronousJobRead"
+        supportedDestinationSyncModes:
```

Comment:
is the `destination` here extraneous?

Reply:
it is (i actually left it out at first), but then I thought we could confuse it with the source syncModes if it's called `supportedSyncModes` so I re-added the destination back in the name...